src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```

recommendation: enum names should better be singular

Reply:
not fixed

Reply:
@karma-o how to rename file in github or i need rename in my computer?